pom.xml

```diff
@@ -188,6 +188,11 @@
             <version>${httpcomponents.version}</version>
         </dependency>
         <dependency>
+            <groupId>org.apache.httpcomponents</groupId>
+            <artifactId>httpasyncclient</artifactId>
+            <version>4.1.2</version>
```

Comment:
actually can you switch to the same version pattern `<version>${httpcomponents.version}</version>` or is this unrelated to the other versions?